client/src/AppBundle/Resources/assets/javascripts/main.js

```diff
@@ -101,7 +101,11 @@ $(document).ready(function () {
   if ($submitButtons !== null) {
     $submitButtons.forEach(function ($el) {
       $el.addEventListener('click', function ($e) {
-        this.classList.add('opg-submit-link--disabled', 'govuk-button--disabled')
+        $e.target.classList.add('opg-submit-link--disabled', 'govuk-button--disabled')
```

Comment:
Better to use `$e.target.classList` here rather than `this` as `this` is not always the same when used within different JavaScript functions.

Example:
```javascript
$el.addEventListener('click', function ($e) {
        // This 'this' will mean the button that is clicked
        this.classList.add('opg-submit-link--disabled', 'govuk-button--disabled')
}

$el.addEventListener('click', ($e) => { // ES6 arrow function  
        // This 'this' always represents the object that defined the arrow function
        this.classList.add('opg-submit-link--disabled', 'govuk-button--disabled')
}
```

More reading on this can be found here: https://www.w3schools.com/Js/js_arrow_function.asp